.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,64 @@
+on: [push, pull_request]
+
+name: Continuous-integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
```

Comment:
and here

Reply:
According to `cargo test --help`
```
--all                                        Alias for --workspace (deprecated)
--workspace                           Test all packages in the workspace
```
I will add `--workspace` flag.

Reply:
Ahh yes, thanks!